tls/s2n_resume.c

```diff
@@ -767,6 +767,32 @@ struct s2n_ticket_key *s2n_find_ticket_key(struct s2n_config *config, const uint
     return NULL;
 }
 
+struct s2n_unique_ticket_key {
+    uint8_t initial_key[S2N_AES256_KEY_LEN];
+    uint8_t info[S2N_AES256_KEY_LEN];
+    uint8_t output_key[S2N_AES256_KEY_LEN];
+};
+
+static S2N_RESULT s2n_resume_generate_unique_ticket_key(struct s2n_unique_ticket_key *key)
```

Comment:
We should include the "why" in code comments. Your docs will get lost and someone will say "why do we have to do this, it seems unnecessary, I'll just delete it".